numpy/_core/_add_newdocs.py

```diff
@@ -2698,6 +2701,17 @@
     >>> np.prod(x.shape) * x.itemsize
     480
 
+    >>> import numpy as np
+    >>> arr = np.arange(100).reshape(10, 10, 1)
+    >>> arr_1 = np.broadcast_to(arr, (10, 10, 1000))
+    >>> arr_2 = np.lib.stride_tricks.sliding_window_view(arr, 5, 0)
+    >>> arr.nbytes
+    800
+    >>> arr_1.nbytes
+    800000
+    >>> arr_2.nbytes
+    2400
```

Comment:
This introduces way too much complexity for very little gain.  If anything at all, just do some slicing like `arr[::2]` or so.